cg/store/models.py

```diff
@@ -315,6 +315,7 @@ class Family(Model, PriorityMixin):
 
     priority = Column(types.Enum(Priority), default=Priority.standard, nullable=False)
     synopsis = Column(types.Text)
+    tickets = Column(types.VARCHAR)
```

Comment:
One thing I was realised after we spoke yesterday is that it won't be possible to do lookups of Families based on tickets without specifying all tickets associated with the family in a comma separated string without using the slower LIKE SQL query.

I'm not sure if this is an issue or not or if this has implications for other code doing such lookups, however. 😊

Reply:
Yes, I figured as much. I don't think speed is an issue here though, since it will mostly be used by functions running on crontabs/timers.